### ISSUE TITLE
A11y_SQL Azure Data Studio_New Terminal_Keyboard:Tab key or shift+tab does not move to link "https://aka.ms/pscore6" is n 

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4179]
“[Check out Accessibility Insights](https://accessibilityinsights.io/)! – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: -A11yMAS;-A11yTCS;-A11ySev1;-BM_SQLAzureDataStudio_Oct2020; -SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;- WCAG2.1.1;-Benchmark;-Keyboard;-AILimited;

**Environment Details:**
Application Name: SQL Azure Data Studio
Version 1.22.1 User Setup
Windows Version: Windows10

**Repro Steps:**

1. Launch Azure Data Studio Insiders application.
2. Tab to server drop down and tab again to button new connection press enter key and add the server details.
3. Tab to view menu item in the menu bar and use down key arrow to select item "terminal"

**Actual:**
On navigating using shift+tab or tab to link https://aka.ms/pscore6 keyboard key tab cannot access it.

**Expected:**
Tab and shift+tab from the first line of command should move to link https://aka.ms/pscore6

**User Impact:**
Keyboard users will not be able to access the link.

**Attachment:**
